sbirez/static/js/services/proposalsvc.js

```diff
@@ -67,7 +79,17 @@ angular.module('sbirezApp').factory('ProposalService', function($http, $window,
 
   var saveProposalData = function() {
     var deferred = $q.defer();
-    $http.patch(PROPOSAL_URI + proposal.id + '/partial/', {'data':JSON.stringify(proposalData)}).success(function(data) {
+    $http.patch(PROPOSAL_URI + proposal.id + '/', {'data':JSON.stringify(proposalData)}).success(function(data) {
```

Comment:
Is it intentional that the /partial/ was removed here?  If the existence of saveCompleteData implies that saveProposalData is supposed to save partial submissions successfully, I'm pretty sure /partial/ is what you want...